### ISSUE TITLE
move wasm-bindgen-futures here?

### ISSUE THREAD

**Issue by dakom** [rustwasm/gloo#88]
Seems to me like "spawn a future" is something that gloo would aim to solve - but the crate already exists under `rustwasm` ... wondering if it should be moved here?

**Comment by fitzgen** [comment]
I don't think it makes sense to move the crate into gloo, but it could certainly make sense to re-export it / its functionality here.
